unix/README.md

```diff
@@ -1,161 +1,106 @@
 # Machine Stats for Unix-like systems
 
-Machine Stats for Linux/Unix leverages [Ansible](https://www.ansible.com/) to gather facts in a cross-platform way.
+[![PyPI](https://img.shields.io/pypi/v/machine-stats)](https://pypi.org/project/machine-stats/)
 
-## Data captured
-
-For *nix based systems, by default, the following metrics are captured from the resources and sent and stored in Tidal Migrations:
-
-```
-Host Name
-FQDN
-IP Addresses
-RAM Allocated (GB)
-RAM Used (GB)
-Storage Allocated (GB)
-Storage Used (GB)
-CPU Count
-Operating System
-Operating System Version
-CPU name
-```
-
-## Requirements
-
-1. **Git**
-
-    ```
-    # Debian/Ubuntu
-    sudo apt-get update
-    sudo apt-get install git
-
-    # Fedora
-    sudo dnf -y update
-    sudo dnf -y install git
-
-    # RHEL/CentOS
-    sudo yum update
-    sudo yum install git
-
-    # macOS (Xcode)
-    xcode-select --install
-
-    # macOS (brew)
-    brew install git
-    ```
+A simple and effective way to gather machine statistics (RAM, Storage, CPU)
+from a server environment as a first layer of a [Tidal Migrations discovery
+process](https://guides.tidalmg.com).
 
-    Or follow the installation instructions on [Git's web
-    site](https://git-scm.com/downloads)
-
-2. **`pipenv`**
-
-    ```
-    # Debian 10+/Ubuntu 20.04+
-    sudo apt-get update
-    sudo apt-get install pipenv
-
-    # Fedora 32+
-    sudo dnf -y update
-    sudo dnf -y install pipenv
-
-    # RHEL 7
-    sudo yum install rh-python38
-    scl enable rh-python38 bash
-    python3.8 -m pip install --user pipenv
-
-    # RHEL 8
-    sudo yum install python3
-    python3 -m pip install --user pipenv
-    ```
-
-    Or follow the installation instructions on [`pipenv` web
-    site](https://pipenv.pypa.io/en/latest/install/#installing-pipenv)
+Machine Stats for Linux/Unix leverages [Ansible](https://www.ansible.com/) to
+gather facts in a cross-platform way.
 
 ## Installation
 
-We are going to install Machine Stats for Unix-like systems in user's `$HOME`
-directory instead of system-wide, so the installation won't affect system
-libraries and packages.
-
-First of all, let's obtain the sources of Machine Stats.
-
-Open your terminal emulator and change the current working directory to your
-`$HOME` directory by running this simple command:
-
-```
-cd 
-```
-
-Let's fetch the sources from our Git repository:
+Install locally in a Python 3 environment:
 
 ```
-git clone https://github.com/tidalmigrations/machine_stats.git
+pip install machine-stats
 ```
 
-All of the code should be downloaded into a `machine_stats` directory, let's
-change into that directory:
+If you are already have `pipx` installed and prefer to use that, you can install Machine Stats as the following:
```

Comment:
```suggestion
If you already have `pipx` installed and prefer to use that (`pipx` installs packages in a virtual environment, so it won't mix with your existing system/user Python packages), you can install Machine Stats as the following:
```

I see our previous thread vanished 😂 

but I think the context you provided there is valuable. For people who do want to use an existing VM and don't want to interfere with existing packages and python installations. Also I corrected a typo in the suggested text I wrote 🙈